appservice/src/deploy/waitForDeploymentToComplete.ts

```diff
@@ -44,7 +44,7 @@ export async function waitForDeploymentToComplete(context: IActionContext, clien
         try {
             logEntries = await retry(
                 async (attempt: number) => {
-                    context.telemetry.properties.getLogEntryAttempt = attempt.toString();
+                    addAttemptTelemetry(context, 'getLogEntry', attempt);
```

Comment:
Would it be too extra to use an enum for methodName?

Reply:
Yeah